WMF Framework/SummaryExtensions.swift

```diff
@@ -1,4 +1,58 @@
 extension WMFArticle {
+    func merge(_ article: WMFArticle) {
+        guard article.objectID != objectID else {
+            return
+        }
+        // merge important keys not set by the summary
+        let keysToMerge = ["savedDate", "placesSortOrder", "pageViews"]
+        // ensure these values still exist with an assertion. is there a way to use these directly?
+        assert([\WMFArticle.savedDate, \WMFArticle.placesSortOrder, \WMFArticle.pageViews].count == keysToMerge.count)
```

Comment:
looks like `let keysToMerge = [#keyPath(WMFArticle.savedDate), #keyPath(WMFArticle.placesSortOrder), #keyPath(WMFArticle.pageViews)]` might work, compiler flags members that don't exist